android/build.gradle

```diff
@@ -49,7 +49,7 @@ android {
         // Gradle doesn't know which variant of purchases-hybrid-common to use when using
         // includeBuild and working with a local copy of purchases-hybrid-common
         // https://developer.android.com/studio/build/build-variants#resolve_matching_errors
-        missingDimensionStrategy 'dependencyVersions', 'latestDependencies', 'unityIAP'
+        missingDimensionStrategy 'apis', 'defaults'
```

Comment:
The comment above seems outdated as well? We only have variants right now in `purchases-android`.